### PR TITLE
fix: remove enr padding

### PR DESCRIPTION
**Pull request by 4rgon4ut**
This PR address prysm bootnode ENR fix. This ENR string contains padding containes padding (`==`), which doesn’t correspond encoding scheme that declared in [EIP-778](https://eips.ethereum.org/EIPS/eip-778#text-encoding).

Jobs:
- Check for padding
- Check that ENR string is valid and can be decoded with `enr-cli`, also checks that all non-empty and non-comment strings are ENR strings

Job triggers on PR to the main branch.

### CODE REVIEW

**Comment by filoozom** [comment]
According to https://github.com/sigp/lighthouse/pull/4530#issuecomment-1696814492, the following ENRs are also invalid:
- `chiado-lighthouse-0`
- `chiado-lighthouse-1`
- `chiado-lodestar-0`

Do we know why?

**Comment by 4rgon4ut** [comment]
> According to [sigp/lighthouse#4530 (comment)](https://github.com/sigp/lighthouse/pull/4530#issuecomment-1696814492), the following ENRs are also invalid:
> 
> * `chiado-lighthouse-0`
> * `chiado-lighthouse-1`
> * `chiado-lodestar-0`
> 
> Do we know why?

They seems to work on my side, I was able to get mentioned error only for `chiado-prysm-0` (used same rust crate to check). 
Node also starting to correctly with such ENRs.

File: .github/workflows/check_bootnodes_enrs.yml [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+            echo "Testing decoding in file: $FILE"
+            while read -r LINE; do
+              if [[ "$LINE" =~ enr: ]]; then
+                ENR_LINE=$(echo "$LINE" | sed -e 's/#.*//' -e 's/^[ \t]*//' -e 's/[ \t]*$//')
+                if [[ "$ENR_LINE" =~ ^- ]]; then
+                  BASE64_STRING=$(echo "$ENR_LINE" | awk -F "enr:" '{print $2}' | awk '{gsub(/"/, "", $0); print}')
+                else
+                  BASE64_STRING=${ENR_LINE#enr:}
+                  BASE64_STRING=$(echo "$BASE64_STRING" | awk '{gsub(/"/, "", $0); print}')
+                fi
```
Comment (filoozom) [inline .github/workflows/check_bootnodes_enrs.yml]: This check should be dependent on the file type, right? Checking if the line starts with a dash only works if we're actually using dashes in `yaml` and not using them in `txt`.

I think this should read all the lines that are not comments or empty and then check for the proper format. Maybe even use a `yaml` decoder to make sure that the file has the correct format?

Comment (4rgon4ut) [reply .github/workflows/check_bootnodes_enrs.yml]: Yes, condition satisfied only for `yaml` files. 
You mean include case when strings in `yaml` file will be not dash-prefixed to not skip them if we will change formatting ?

**Review by filoozom — APPROVED** [review]

**Pull request merged**